### DISCUSSION TITLE
OSquery agents on both windows 10 and Ubuntu 18.04 not reflecting on fleet

### DISCUSSION THREAD

**Discussion by Guta-lang** [Security-Onion-Solutions/securityonion#2621] [Unsupported Versions]
I have installed osquery on both hosts, and ran so-allow on the manager. Unfortunately, I'm unable to see these hosts on Kolide fleet. I have built a fleet node, it's not within the manager. My set-up is in a production environment.

Anyone who's been able to deploy this successfully?

**Comment by innovate-support** [comment]
I had similar issue.  My fix was to manually edit my launcher.flags config file that got auto created when installing on Win10.  Not sure if you're having the same issue I was, but see my solution at the bottom of this discussion.
https://github.com/Security-Onion-Solutions/securityonion/discussions/2317

**Reply by Guta-lang** [reply]
Thank you for your input. I had already seen your solution from the discussion and tried it, but unfortunately it didn't work on my side.

**Comment by defensivedepth** [comment]
@Guta-lang 

Review the `Application` eventlogs in the Windows 10 host - there should be logs from osquery/Launcher. Can you please post any errors that you see?

**Reply by Guta-lang** [reply]
"caller=log.go:124 ts=2021-01-15T06:43:07.5571447Z caller=level.go:63 level=info caller=updater.go:102 msg="updater interrupted" err="enrolling host: transport error in enrollment: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp 192.168.0.231:8090: i/o timeout\""
"

**Reply by defensivedepth** [reply]
@Guta-lang is 192.168.0.231 the IP of your Security Onion Manager or Fleet Node?

**Reply by Guta-lang** [reply]
@defensivedepth 192.168.0.231 is the IP of the Fleet Node

**Reply by defensivedepth** [reply]
Do you get any errors when running the following on the Fleet standalone node?

`sudo salt-call state.highstate`

Please paste the output of the following command, run from the Manager (sanitizing as needed):

`sudo salt-call pillar.get global`

**Reply by Guta-lang** [reply]
There are no any errors when running sudo salt-call state.highstate on the fleet node

**Reply by Guta-lang** [reply]
Output from sudo salt-call pillar.get global

local:
    ----------
    airgap:
        False
    cortexkey:
        xxxxx
    cortexorgname:
        SecurityOnion
    cortexorguserkey:
        xxxxx
    cortexplaysecret:
        xxxxx
    dockernet:
        172.17.0.0
    fleet_custom_hostname:
        None
    fleet_hostname:
        sosf
    fleet_ip:
        192.168.0.231
    fleet_manager:
        True
    fleet_node:
        True
    fleet_packages-timestamp:
        2021-01-17-13:33
    fleet_packages-version:
        6
    hivekey:
       xxxxx
    hiveplaysecret:
       xxxxx
    hnmanager:
        xxxxx
    ids:
        Suricata
    imagerepo:
        securityonion
    managerip:
        xxxxx
    managerupdate:
        1
    mdengine:
        ZEEK
    ntpserver:
    pipeline:
        redis
    proxy:
    sensoronikey:
        xxxxx
    soversion:
        2.3.21
    url_base:
       xxxxx
    wazuh:
        1

**Reply by defensivedepth** [reply]
Please run the following command on both the Manager & the Fleet node and post the Exposed Ports section for both of them:

`sudo docker inspect so-nginx | jq '.[0] | {Config}'`

**Reply by Guta-lang** [reply]
For the Manager;
"ExposedPorts": {
      "443/tcp": {},
      "80/tcp": {},
      "8090/tcp": {}

**Reply by Guta-lang** [reply]
For the Fleet node;
"ExposedPorts": {
      "443/tcp": {},
      "80/tcp": {},
      "8090/tcp": {}

**Reply by Guta-lang** [reply]
@defensivedepth Are there any updates on this problem?

**Reply by defensivedepth** [reply]
Can you confirm that your endpoints have a route to `192.168.0.231` ?
